### ISSUE TITLE
A11y: Add accessibility conformance report and VPAT to site

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/clean-and-green-philly#698]
## Describe the task
@marvieqa drafted our Accessibility Statement and Accessibility Conformance Report (also known as Voluntary Product Accessibility Template (VPAT)) and we need to host them on the site.

## Acceptance Criteria
- [ ] Add a link for "Accessibility Statement" to our footer
- [ ] That link should go to a new page at  https://www.cleanandgreenphilly.org/accessibility-statement. This new page should include the contents of the accessibility statement (see "Resources" below)
- [ ] At the bottom of the Accessibility Statement there is this line:
`You can find our full evaluation report in VPAT®️ format. (link)` That link should go to another new page at https://www.cleanandgreenphilly.org/vpat. We may find a way to do a download as PDF later on down the line, but we would have to test the PDF for accessibility first, so for now just create a new page with the VPAT contents (see "Resources" below). Note: the tables on the VPAT should be coded as tables and not divs.

## Resources
Accessibility Statement - https://docs.google.com/document/d/1WBmofwiaW-BLc27z-Qek7HSfc9iuoCWAWU7JNLwAgsc/edit#heading=h.y3te9fvbiauq
VPAT - [CleanAndGreenPhilly_VPAT_05312024.pdf](https://github.com/user-attachments/files/15892296/CleanAndGreenPhilly_VPAT_05312024.pdf)

**Comment by zheyichn** [comment]
Hi! May I be assigned this ticket pls?

**Comment by bacitracin** [comment]
@zheyichn Yep you got it

**Comment by zheyichn** [comment]
Hi @CodeWritingCow I have a component design question regarding the /vpat page. The content within the [pdf](https://github.com/user-attachments/files/15892296/CleanAndGreenPhilly_VPAT_05312024.pdf) contains multiple tables, I was thinking about extract one Table component. But there are different stylings (underlie, italic, bold) applied on different rows within each table, which makes reusing the same Table component challenging. Do you think it would be better to stick with a single dedicated VPAT page component? Thank you.

**Comment by CodeWritingCow** [comment]
Hi @zheyichn, if you want to use a reusable Table component, consider using `<Table>` from the `@nextui-org/react` library. We are using it in this codebase (ex. `src/components/PropertyDetailSection.tsx`).

Within the VPAT page component, you can create multiple `<Table>`s to match the PDF's tables. From there, you should be able to stylize each table's content. Let me know if you have questions!

**Comment by bacitracin** [comment]
@zheyichn @CodeWritingCow  Hey just chiming in here to say that we're not wedded to using the NextUI library. Many of the components from that library have known accessibility problems, and we want to empower devs to write their own components from scratch if they're not a11y-friendly. 

I haven't tested NextUI's Table component, but given the existence of the multiple subheaders in the VPAT tables, I would suggest writing them with a plain `<table>` tag. Break them out into as many individual tables as you need, and I don't think it needs to be reusable. There may need to be a lot of customization on these tables to make them sound decent in a screenreader, and NextUI/trying to make it reusable adds an extra layer I don't think is worth it.

Here's some more info about accessible tables:
- https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced
- https://www.w3.org/WAI/tutorials/tables/tips/

**Comment by CodeWritingCow** [comment]
@bacitracin thanks for your recommendation. I didn't realize that NextUI has accessibility problems. While I like the idea of using a reusable component to avoid code repetition, the tradeoffs might not be worth it in this case.

@zheyichn Tracy is much more knowledgeable about accessibility, so I'd defer to her on how to approach this ticket. So let's use regular `<table>` tags.

Once the VPAT page is built, we can always look again into whether it can be refactored to use reusable components.
